docs/reference/module-api/module-overview.md

```diff
@@ -838,19 +838,82 @@ module.exports = {
 };
 ```
 
-### `queries(self)`
+### `queries(self, query)`
+
+The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that will be called (`query`). It must return an object. That object can have two properties:
+<!-- TODO: Link to a real guide on using queries when available. This is simply reference. -->
+
+| `queries` properties | Description |
+| ---- | ---- |
+| [`builders`](#builders) | An object of "builders," or chainable methods, that refine the query or otherwise change its behavior prior to execution. |
+| [`methods`](#methods) | An object of method functions that are used to execute some action on the query. |
+
+#### `builders`
+
+Query builders are defined as objects with a set of specific properties available to them. Builders often take an argument or have a default value that is used instead.
+<!-- TODO: Link to a reference of core builders when available. -->
+
+| Builder properties | Description |
+| ---- | ---- |
+| `def` | The default value for the builder. |
+| `launder` | A function used to validate values passed as arguments when `applyBuildersSafely` is called on the query. Returns `true` if valid.<br />This is required to use the builder in a REST API query string. |
+| `choices` | A function returning an array of "choice" objects with `value` and `label` properties. This is returned from the `toChoices` query method if set. |
+| `set` | A function to called when the builder is invoked, instead of the default `query.set` method. This should usually include running the `query.set` itself in addition to other work. |
```

Comment:
```suggestion
| `set` | A function called when the builder is invoked instead of the default `query.set` method. This usually includes running the `query.set` itself. |
```
I remove "to", assuming the original intent was "to call". I removed "in addition to other work" because it implied by "should usually include", which is in turn "usually includes"